File: elucidate-common-lib/src/test/java/com/digirati/elucidate/common/service/impl/IRIBuilderServiceImplTest.java
package com.digirati.elucidate.common.service.impl;

import java.util.Date;

import org.junit.Test;

import static org.junit.Assert.assertNotNull;

public class IRIBuilderServiceImplTest {

    @Test
    public void testToString() {
        Date since = new Date();
        String dateString = IRIBuilderServiceImpl.toString(since);
        System.out.println(dateString);
        assertNotNull(dateString);
    }
}

File: elucidate-common-lib/src/test/java/com/digirati/elucidate/common/service/impl/IRIBuilderServiceImplTest.kt
package com.digirati.elucidate.common.service.impl

import org.junit.Assert.assertNotNull
import org.junit.Test
import java.util.*

class IRIBuilderServiceImplTest {
    @Test
    fun testToString() {
        val since = Date()
        val dateString = IRIBuilderServiceImpl.toString(since)
        println(dateString)
        assertNotNull(dateString)
    }
}